tla-pp/src/test/scala/at/forsyte/apalache/tla/pp/TestNormalizer.scala

```diff
@@ -63,4 +87,32 @@ class TestNormalizer extends FunSuite with BeforeAndAfterEach {
         .typed(types, "b")
     assert(expected == output)
   }
+
+  test("""x => y ~~> ~x \/ y""") {}
+
+  test("""~ (x => y) ~~> x /\ ~y""") {}
+
+  test("""(~FALSE <=> x) ~~> TRUE <=> x""") {}
+
+  test("""~ (~FALSE <=> x) ~~> TRUE \neq x""") {}
+
+  test("""~ \E x \in s . y ~~> \A x \in s . ~y""") {}
+
+  test("""~ \A x \in s . y ~~> \E x \in s . ~y""") {}
+
+  test("""~ (x < y) ~~> x >= y""") {}
+
+  test("""~ (x <= y) ~~> x > y""") {}
+
+  test("""~ (x > y) ~~> x <= y""") {}
+
+  test("""~ (x >= y) ~~> x < y""") {}
+
+  test("""~(x \neq y) ~~> x <=> y""") {}
+
+  test("""~(~FALSE \in s) ~~> ~(TRUE \in s)""") {}
+
+  test("""~<>x ~~> []~x""") {}
+
+  test("""~[]x ~~> <>x""") {}
```

Comment:
I believe there is a problem in the implementation of this normalization scenario. I wrote this description `~[]x ~~> <>x` description based on implementation, but it is not a valid equivalence. It should be `~[]x ~~> <>~x`. The implementation is currently:
```scala
    case ex @ OperEx(TlaTempOper.box, args @ _*) =>
      if (!neg) {
        OperEx(TlaTempOper.box, args map nnf(true): _*)(ex.typeTag)
      } else {
        OperEx(TlaTempOper.diamond, args map transform: _*)(ex.typeTag)
      }
```
and I believe it was meant to be:
```scala
    case ex @ OperEx(TlaTempOper.box, args @ _*) =>
      if (!neg) {
        OperEx(TlaTempOper.box, args map transform: _*)(ex.typeTag)
      } else {
        OperEx(TlaTempOper.diamond, args map nnf(true): _*)(ex.typeTag)
      }
```
https://github.com/informalsystems/apalache/blob/normalizer-tests/tla-pp/src/main/scala/at/forsyte/apalache/tla/pp/Normalizer.scala#L160

@konnov @Kukovec @shonfeder Can you verify this, please?